lib/core/macros.nim

```diff
@@ -470,6 +470,25 @@ proc bindSym*(ident: string | NimNode, rule: BindSymRule = brClosed): NimNode {.
   ## returned even if the symbol is not ambiguous.
   ##
   ## See the `manual <manual.html#macros-bindsym>`_ for more details.
+  ## 
+  ## See also `dynamicBindSym`_.
+
+proc dynamicBindSym*(ident: string | NimNode, rule: BindSymRule = brClosed): NimNode {.
```

Comment:
currently `dynamicBindSym` have notorious bug. if you pass an expression other than simple variable, the parameter is not evaluated by the VM. see #11496.

it would be great if you can also fix that issue by evaluating the parameters before calling dynamicBindSym.

Reply:
I tried doing what you said in a couple ways, but it didn't work.

This isn't really a huge issue anyway, all one has to do is `let name = ...; bindSym(name)` instead of `bindSym(...)`. What I can do is make a `dynamicBindSymImpl` proc which binds to the compiler then make `dynamicBindSym` a proc which calls that, which will fix the issue since `dynamicBindSym`'s parameter is like the `name` in `let name`.